misk-jobqueue-testing/src/main/kotlin/misk/jobqueue/FakeJobQueue.kt

```diff
@@ -93,54 +91,88 @@ class FakeJobQueue @Inject constructor(
   fun handleJobs(
     queueName: QueueName,
     assertAcknowledged: Boolean = true,
-    retries: Int = 1
+    retries: Int = 1,
+    considerDelays: Boolean = false
```

Comment:
I’d love to flip this default, or retire this option. You’re right that delays should just work.

Reply:
This will very probably break a bunch of tests in Franklin :)) 
It is better to do in a separate PR:
1) Introduce an option in FakeJobQueueModule
2) Use this option in all services and fix the tests
3) Then make this option default